nixos/modules/services/databases/mysql.nix

```diff
@@ -93,6 +93,13 @@ in
         description = "Location of the file which stores the PID of the MySQL server";
       };
 
+      enableSocketAuth = mkEnableOption ''Whether to let root login as the mysql root
```

Comment:
This will render as `Whether to enable Whether to let....`

Reply:
ACK, changed ;)